lib/ex_doc.ex

```diff
@@ -25,6 +25,7 @@ defmodule ExDoc do
       project: nil,
       retriever: ExDoc.Retriever,
       source_beam: nil,
+      source_ref: "master",
```

Comment:
Ok, I guess if we ever need to find out what was passed by the user, we can add it later to it